### DISCUSSION TITLE
Sponsor Connect & pre-releases

### DISCUSSION THREAD

**Discussion by jfraschilla** [PrismLibrary/Prism#2344] [General]
I have a question about GitHub sponsorship and access to WPF Prism pre-releases.  I currently sponsor Brian at $5 month but on his sponsor link to does not mention which support level get you SponsorConnect access.  On Dan's sponsor page it says $10 per month get you access.  I currently sponsor Brian because I am more interested in WPF Prism applications and I follow Brian on YouTube.  Do I need to sponsor Brian at $10/month for pre-release and will that give me access to the SponsorConnect NuGet pre-release packages?  Also, there seems to be a problem when I click on the SponsorConnect link on the WPF package.  I get a HTTP ERROR 404.

**Comment by dansiegel** [comment]
Sponsor Connect v2 went live this past week. This is kind of a big deal for those who are sponsoring @brianlagunas as it now provides some of the great benefits that my sponsors have had such as the Sponsor Connect Discord and the Sponsor Connect package feed with the Prism CI packages. Please note that there is a package filter in play so you may see packages that you are not authorized to download. For instance only my Sponsors have access to Prism.Magician.

Please note that you must sponsor Brian and/or myself at a $10/month minimum to access the Discord and Prism CI Packages. However, we do ask that if:

- You are financially able
- Prism has helped you ship amazing apps
- The content the Brian and I try to put out for the community has helped your career or your business

Please consider a higher level -OR- sponsor both of us and help support us in being able to provide more back to the community. 

If you're having difficulty with the package feed or anything else the best thing is to join us on the Sponsor Connect Discord and ask your question.

**Reply by jfraschilla** [reply]
I am unable to connect to https://sponsorconnect.dev.  I get a spinner that says "Loading..."

**Reply by dansiegel** [reply]
Please try clearing your cache and refreshing the page.

**Reply by jfraschilla** [reply]
Still have problem after clearing cache

**Reply by jfraschilla** [reply]
I tried again on 2/21 and was initially successful.  I got the first couple of pages to allow privilege's and to register.  I received a confirmation email.  However, when I click on the link in the email it tries to reconnect to https://sponsorconnect.dev and then I get the spinning "Loading..." message

**Comment by dansiegel** [comment]
Sorry for the delayed response @jfraschilla as I was away on some personal business... can you email dsiegel @ avantipoint dot com so that I can try to work with you to resolve whatever problem that you're experiencing.

**Comment by thomasgalliker** [comment]
@dansiegel is https://sponsorconnect.dev/ still maintained? I mean, do I get nuget pre-releases of Prism.DryIoc.Maui there? I haven't seen any updates for some time.

**Reply by brianlagunas** [reply]
You must purchase a commercial plus license in order to gain access to our private builds.

**Reply by dansiegel** [reply]
Commercial Plus actually as the Commercial License only gives you access to what's on NuGet.org

**Comment by jfraschilla** [comment]
@dansiegel, Where can we find more details about the features in the Commercial Plus subscription?  In particular, what MAUI features are available that are not in the Free version?
